Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -15,18 +15,19 @@
  */
 
 #import "FIRPhoneAuthProvider.h"
+#import "FirebaseAuth_Internal.h"
 
-#import "FIRLogger.h"
+#import FirebaseCore(FIRLogger.h)
```

Comment:
Why the FirebaseCore macro? I can see that it makes import sources clearer. Any references about this style?